tango_simlib/tests/test_sim_sdd_xml_parser.py

```diff
@@ -1,19 +1,19 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
 #########################################################################################
 # Author: cam@ska.ac.za                                                                 #
 # Copyright 2018 SKA South Africa (http://ska.ac.za/)                                   #
 #                                                                                       #
 # BSD license - see LICENSE.txt for details                                             #
 #########################################################################################
+from __future__ import absolute_import, division, print_function
+from future import standard_library
+standard_library.install_aliases()
```

Comment:
@amakhaba This requires a `#noqa` see https://github.com/ska-sa/katcp-python/blob/4240fb45ec8cb73c4b60490e6e43ffe97700f567/doc/conf.py#L17